parcelgen.py

```diff
@@ -55,7 +55,7 @@ def member_map(self):
 
     def gen_getter(self, typ, member):
         method_name = ""
-        if typ == "boolean" and member.startswith("is"):
+        if typ == "boolean" and (member.startswith("is") or member.startswith('has')):
```

Comment:
It looks like most of this file uses " instead of '.  Can we keep that consistent here?